Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,6038 @@
+        args.update(read_json_policy(json_template_id, name))
+    validate_server_policy(client.version, args)
+    response = client.server_policy_create_request(
+        name=args['name'],
+        deployment_mode=args['deployment_mode'],
+        virtual_server=args['virtual_server'],
+        server_pool=args.get('server_pool'),
+        protected_hostnames=args.get('protected_hostnames'),
+        client_real_ip=args.get('client_real_ip'),
+        syn_cookie=args.get('syn_cookie'),
+        half_open_thresh=args.get('half_open_thresh'),
+        http_service=args.get('http_service'),
+        https_service=args.get('https_service'),
+        http2=args.get('http2'),
+        proxy=args.get('proxy'),
+        redirect_to_https=args.get('redirect_to_https'),
+        inline_protection_profile=args.get('inline_protection_profile'),
+        monitor_mode=args.get('monitor_mode'),
+        url_case_sensitivity=args.get('url_case_sensitivity'),
+        comments=args.get('comments'),
+        mach_once=args.get('mach_once'),
+        allow_list=args.get('allow_list'),
+        replace_msg=args.get('replace_msg'),
+        scripting=args.get('scripting'),
+        scripting_list=args.get('scripting_list'),
+        retry_on=args.get('retry_on'),
+        retry_on_cache_size=arg_to_number(args.get('retry_on_cache_size')),
+        retry_on_connect_failure=args.get('retry_on_connect_failure'),
+        retry_times_on_connect_failure=arg_to_number(args.get('retry_times_on_connect_failure')),
+        retry_on_http_layer=args.get('retry_on_http_layer'),
+        retry_times_on_http_layer=arg_to_number(args.get('retry_times_on_http_layer')),
+        retry_on_http_response_codes=argToList(args.get('retry_on_http_response_codes')),
+        certificate_type=args.get('certificate_type'),
+        lets_certificate=args.get('lets_certificate'),
+        multi_certificate=args.get('multi_certificate'),
+        certificate_group=args.get('certificate_group'),
+        certificate=args.get('certificate'),
+        intergroup=args.get('intergroup'),
+        ip_range=args.get('ip_range'),
+    )
+    command_results = generate_simple_command_results(object_type=OutputTitle.SERVER_POLICY.value,
+                                                      object_name=name,
+                                                      action=OutputTitle.CREATED.value,
+                                                      response=response)
+
+    return command_results
+
+
+def server_policy_update_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """Update a server policy.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    name = args['name']
+    if json_template_id := args.get('json_template_id'):
+        args = read_json_policy(json_template_id, name)
```

Comment:
just to make sure, you're reassiging args here on purpose?

Reply:
Yes, I read the args from JSON file.